inform7/extensions/basic_inform/Sections/Glulx and Glk.w

```diff
@@ -133,6 +133,9 @@ Definition: a glk event is dependent on the player rather than independent of th
 To set the/-- glk event type to (t - glk event):
 	(- SetGlkEventType({t}); -).
 
+To say current line input of (w - glk window):
```

Comment:
I added this phrase as it is more efficient when joining texts together. Now I'm wondering if we need the next phrase, which manually copies the buffer to a new text. It could probably be removed.